features/loopMessages/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * ------------------------------
+ * Loop Messages
+ * ------------------------------
+ * The bot will keep a message as the last one in a channel, with an interval between each post,
+ * to keep important information more visible to users
+ *
+ * How it works:
+ * At startup and repeatedly by a given interval, for each loop message, the bot will:
+ * 1. Get the target channel of the loop message
+ * 2. Fetch the last 50 messages of the channel
+ * 3. If the last message is already the message we want to loop, do nothing (we don't want to spam the channel)
+ * 4. Delete all loop messages (so the channel isn't filled with bot messages)
+ * 5. Send the loop message
+ */
+
+const loopMessages = require("./messages.js");
+
+// time between each check to send a new loop message
+const INTERVAL_TIME = 5 * 60 * 1000; // 5 minutes (ms)
```

Comment:
Is 5 minutes an appropriate amount of time? Should it be longer? I'm not sure if we want this to be customization depending on the loop message.

Reply:
I am not sure, to be honest. The bot won't keep sending new messages if the last one is from it so a low interval time won't create spam, but the messages on #jobs aren't that active to hide the bot message if the interval time is too big. :thinking:

Reply:
> I'm not sure if we want this to be customization depending on the loop message.

The initial implementation had an interval field on each message, but I decided to remove it to avoid making it more complex than it is initially needed. We can add when we feel necessary.

Reply:
I increased the interval to 40 minutes

Reply:
ok now each message can have a different interval